docs/quantum_chess/quantum_chess_rest_api.ipynb

```diff
@@ -784,7 +784,6 @@
     "\n",
     "\n",
     "app = Flask(__name__)\n",
-    "run_with_ngrok(app)\n",
```

Comment:
What's going on here?  It looks like this will prevent the app from running.

Reply:
We removed ngrok imports in the last update, but missed this line. We're using werkzeug for local hosting instead.